.github/workflows/integration-build-product.yml

```diff
@@ -252,12 +274,25 @@ jobs:
           rm -rf simulator_artifacts ci_artifacts
           mkdir platform_configs
           source ci_scripts/bin/activate
-          integration-pipeline raise
+          # Check if a PR triggered the workflow
+          pip install bump2version
```

Comment:
should bump2version already be in our container image ?

Reply:
it current is not, but maybe could be added

Reply:
let's add it then instead of reinstalling it 10 times a day